web3/index.js

```diff
@@ -89,3 +95,17 @@ function loadScripts () {
   });
 }
 
+function configureApi (config) {
+  const { DAPPS } = config;
+
+  const dappsInterface = DAPPS.split(':')[0];
+  const dappsPort = DAPPS.split(':')[1];
+
+  if (dappsInterface) {
+    window.secureApi._dappsInterface = dappsInterface;
```

Comment:
Yeah. Let's use proper public method for new versions and that hack for old ones (i.e. if the method is not available)